contributing.md

```diff
@@ -45,5 +50,49 @@ include tests and documentation, and keep in mind backward-compatibility,
 portability, as well as the impact on memory usage and performance.
 
 See [getting started](basics/getting_started.html) for learning how to work on the
-code base, and the [patch acceptance process](basics/patching.html) for sending
-your contribution.
+code base.
+
+### Patch Acceptance Process
+
+1. Read the [Contributor policy and governance plan](governance.html).
+1. Discuss your plan and design, and get agreement on a GitHub issue or our
+   [mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
+1. PRs that change or add behavior are not accepted without being tied to an
+   issue.
+1. Significant changes need a [design document](/designs/index.html).
+1. Prepare a git commit that implements the feature. Don't forget to add tests
+   and update the documentation.
+1. Ensure you've signed a [Contributor License
+   Agreement](https://cla.developers.google.com).
+1. Send us a pull request on
+   [GitHub](https://github.com/bazelbuild/bazel/pulls). If you're new to GitHub,
+   read [about pull
+   requests](https://help.github.com/articles/about-pull-requests/). Note that
+   we restrict permissions to create branches on the main Bazel repository, so
+   you will need to push your commit to [your own fork of the
+   repository](https://help.github.com/articles/working-with-forks/).
+1. Wait for a Bazel team member to assign you a reviewer. It should be done in 2
+   business days (excluding holidays in the USA and Germany). If you do not get
+   a reviewer within that time frame, you can ask for one by sending a mail to
+   [bazel-dev@googlegroups.com](mailto:bazel-dev@googlegroups.com).
+1. Work with the reviewer to complete a code review. For each change, create a
+   new commit and push it to make changes to your pull request. If the review
+   takes too long (e.g. reviewer is unresponsive), please also send an email to
+   [bazel-dev@googlegroups.com](mailto:bazel-dev@googlegroups.com).
+1. A Bazel maintainer at Google will apply the patch to our internal version
+   control system.
+
+   This triggers [internal presubmit checks](https://bazel.build/governance.html#are-you-done-open-sourcing-bazel)
```

Comment:
That link should be removed.
It does does not point to a dashboard of description of the actual tests.
Instead it points to a reason why we have internal tests. This is both unhelpful and unsatisfying.

Reply:
+1 to removing the link, especially as we moved that to the FAQ.

Reply:
Done.